Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -89,7 +89,7 @@ public override void Shutdown()
             return true;
         }
 
-        if (!input.StartsWith(RadioChannelPrefix))
+        if (!(input.StartsWith(RadioChannelPrefix) || input.StartsWith(RadioChannelAltPrefix)))
```

Comment:
doesn't it work without first bracket you added?

Reply:
Probably, I just have a habit of using brackets liberally. I'll remove it.

Reply:
Actually it doesn't, the ! only applies to the first input.StartsWith. I'll leave the brackets there.

Reply:
> Actually it doesn't, the ! only applies to the first input.StartsWith. I'll leave the brackets there.

Ah, I read the code wrongly mb